### PR TITLE
feat: JSON-LD credential format support

### PR DESCRIPTION
**Pull request by sairanjitAW**
- Add w3c credential details screen
- Add rn-nodify package to hack the packages dependent on the crypto

### CODE REVIEW

File: app/screens/CredentialDetailsW3C.tsx [hunk 1]
```diff
@@ -107,55 +115,65 @@ const CredentialDetailsJSONLD: React.FC<CredentialDetailsProps> = ({ navigation,
...
     OCABundleResolver.resolveAllBundles(params).then(bundle => {
       setOverlay({ ...overlay, ...bundle })
     })
-  }, [credential])
+  }, [w3cCredential])
 
-  const handleOnRemove = () => {
-    setIsRemoveModalDisplayed(true)
-  }
+  // const handleOnRemove = () => {
```
Comment (nbAmit) [inline app/screens/CredentialDetailsW3C.tsx:162]: Please remove this

File: app/screens/CredentialDetailsW3C.tsx [hunk 2]
```diff
@@ -48,7 +56,7 @@ const CredentialDetailsJSONLD: React.FC<CredentialDetailsProps> = ({ navigation,
     cardLayoutOverlay: undefined,
   })
 
-  const credentialConnectionLabel = credential.credential.name
+  // const credentialConnectionLabel = credential.credential.issuerId
```
Comment (nbAmit) [inline app/screens/CredentialDetailsW3C.tsx:59]: Please remove this

Comment (sairanjitAW) [reply app/screens/CredentialDetailsW3C.tsx:59]: removed

File: app/screens/CredentialDetailsW3C.tsx (hunk 1 above)
Comment (sairanjitAW) [reply app/screens/CredentialDetailsW3C.tsx:162]: removed

**Pull request merged**